client/src/components/Flag.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+
+export default function Flag() {
+  return (
+    <div id="flag-container">
+      <div className="flag" id="flag1"></div>
```

Comment:
Going off of the previous two comments, you could then use the visability prop to replace lines 6-15 with a single map like this:
```
{visiblities.map((visability) => <div className="flag" id={visability.id} visability=visablity.value></div>
```